docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -5,23 +5,59 @@ weight: 50
 type: "docs"
 ---
 
-This example shows how to wire [Kubernetes cluster events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#event-v1-core),
-using the API Server Source, for consumption by a function that has been implemented as a Knative Service.
+This example shows how to wire
+[Kubernetes cluster events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#event-v1-core),
+using the API Server Source, for consumption by a function that has been
+implemented as a Knative Service.
 
 ## Before you begin
 
-1. You must have a Knative cluster running both the Serving and Eventing components.
-   To learn how to install the required components, see [Installing Knative](../../../install).
+1. You must have a Knative cluster running both the Serving and Eventing
+   components. To learn how to install the required components, see
+   [Installing Knative](../../../install).
 1. You can follow the steps below to create new files, or you clone a copy from
    the repo by running:
 
    ```shell
-   git clone -b "release-0.13" https://github.com/knative/docs knative-docs
+   git clone -b "release-0.14" https://github.com/knative/docs knative-docs
```

Comment:
We should use a release shortcode here, but I'll do a pass later once the website build is cleaned up.